lib/index.js

```diff
@@ -167,11 +167,11 @@ bedrock.events.on('bedrock-passport.authenticate', function(info, callback) {
       // handle `generateResource` feature
       var roles =
         bedrock.jsonld.getValues(info.user.identity, 'sysResourceRole');
-      roles.forEach(function(role) {
-        if(role.generateResource === 'id') {
-          role.resource = [identity.id];
+      for(var i = 0; i < roles.length; i++) {
```

Comment:
It really doesn't matter that much, but we've had a style of using `++i` rather than `i++` in loops just about everywhere (unless there's an actual reason to do it differently).